src/components/Header/components/Logo/index.jsx

```diff
@@ -0,0 +1,17 @@
+import { Link } from 'react-router-dom'
+import logo from '/src/assets/header/logo.svg'
+import subText from '/src/assets/header/subText.svg'
+import styles from './logo.module.scss'
+
+function Logo() {
+  return (
+    <div>
+      <Link className={styles.logo} to="/">
+        <img src={logo} alt="Logo" />
```

Comment:
use span/div like wrapper for images or svg's
![image](https://github.com/user-attachments/assets/c634d3e5-0cf8-4f86-a4f7-2bea24350e66)